wallet/src/repository/wallet/mod.rs

```diff
@@ -1659,6 +1660,37 @@ where
         Ok(!(state.transient_internal_addresses.is_empty()
             && state.transient_external_addresses.is_empty()))
     }
+
+    pub fn export_master_key(&self, password: &[u8]) -> Result<String> {
```

Comment:
We should always use the `Password` type, is there any reason to use `&[u8]` here?

```suggestion
    pub fn export_master_key(&self, password: types::Password) -> Result<String> {
```

Reply:
very true